src/neo/Ledger/Blockchain.cs

```diff
@@ -442,9 +369,6 @@ protected override void OnReceive(object message)
                 case FillMemoryPool fill:
                     OnFillMemoryPool(fill.Transactions);
                     break;
-                case Header[] headers:
-                    OnNewHeaders(headers);
-                    break;
```

Comment:
If we received headers, we just ignore them? I don't think it's a good idea. Why not we retain the old logic?

Reply:
In the old logic, we will save the headers. But I think it's useless.

Reply:
Maybe we can make it compatible to get blocks by both index and hash?

Reply:
We retain the `MessageCommand. GetBlocks`,  other nodes could use it to get blocks by hash.